distributed/comm/tests/test_comms.py

```diff
@@ -769,6 +769,10 @@ async def test_tcp_comm_closed_explicit(tcp):
     await check_comm_closed_explicit("tcp://127.0.0.1")
 
 
+@pytest.mark.xfail(
+    sys.version_info[:2] == (3, 7),
+    reason="This test fails on python 3.7 with certain versions of openssl",
+)
```

Comment:
Not meant to block merging, but we could make this `xfail` a bit more specific with a `if tcp is distributed.comm.tcp` check inside the body of this test. Not a huge deal either way, I'll defer to you if you think it's worth it

Reply:
If you do that the test never runs, `pytest.xfail` is functionally the same as `pytest.skip`. I'd rather be a bit too broad here, since we'll be deleting all this soon anyway.